docs/nifgen/rep_caps.rst

```diff
@@ -87,4 +87,30 @@ markers
         passes a string of :python:`'Marker0, Marker1, Marker2'` to the set attribute function.
 
 
+data_markers
+------------
+
+    .. py:attribute:: nifgen.Session.data_markers[]
+
+        If no prefix is added to the items in the parameter, the correct prefix will be added when
+        the driver function call is made.
+
+        .. code:: python
+
+            session.data_markers['0-2'].channel_enabled = True
+
+        passes a string of :python:`'DataMarker0, DataMarker1, DataMarker2'` to the set attribute function.
```

Comment:
This is very bad documentation, because it documents what the implementation does under the hood which is irrelevant to the Python programmer.

We should (separate PR naturally) rewrite this to be relevant to the user. @ni-jfitzger  want to tackle it? I can help wordsmith.

Reply:
Yeah, I'm not thrilled with this documentation easier, but for a different reason. I don't like the use of channel_enabled in the example code. I understand that it just an example attribute which isn't necessarily supported. It might be confusing to users, though.

I don't mind tackling the improvement of this documentation, but I'll have to do some digging to figure out how, since it's generated.

Reply:
It should just be a matter of updating [rep_caps.rest.mako](https://github.com/ni/nimi-python/blob/5be94c69ca4677d8062f2b84e5f3b2f1136edbe9/build/templates/rep_caps.rst.mako#L42)

Reply:
Related issues, for which there appears to be a fair bit of duplication:
#1388
#1338
#1334
#1126

#1351 would be a nice one to fix too.